orangecontrib/imageanalytics/widgets/owimageembedding.py

```diff
@@ -116,7 +116,7 @@ def _init_server_connection(self):
         )
 
     def set_data(self, data):
-        if data is None:
+        if data is None or not len(data):
```

Comment:
My comment was regarding the same check as in the `commit` method (zero length of `self._image_attributes`). Below the changed if statement here is a similar check where the _"..., but without image attributes."_ text is displayed.